packages/fx-core/resource/package.nls.json

```diff
@@ -670,5 +670,15 @@
   "error.deploy.AzureStorageSetContainerPropertiesError.Notification": "Unable to set properties of container '%s' in Azure Storage Account '%s' due to error: %s. Refer to the [Output panel](command:fx-extension.showOutputChannel) for more details.",
   "error.deploy.AzureStorageSetContainerPropertiesError": "Unable to set properties of container '%s' in Azure Storage Account '%s' due to error: %s. The error responses from Azure are:\n %s. \nIf the error message specifies the reason, fix the error and try again.",
   "error.core.failedToLoadManifestId": "Unable to load manifest id from path: %s. You must run provision first.",
-  "error.core.appIdNotExist": "Cannot find app id: %s. Maybe your current M365 account does not have permission, or the app has alredy been deleted."
+  "error.core.appIdNotExist": "Cannot find app id: %s. Maybe your current M365 account does not have permission, or the app has alredy been deleted.",
+  "driver.apiKey.description.create": "Create an API key on Developer Portal for authentication in Open API spec.",
+  "driver.aadApp.apiKey.title.create": "Creating API key...",
+  "driver.apiKey.log.startExecuteDriver": "Executing action %s",
+  "driver.apiKey.log.skipCreateApiKey": "Environment variable %s already exists. Skip creating API key.",
+  "driver.apiKey.log.apiKeyNotFound": "Environment variable %s exists but failed to retrieve API key from Developer Portal. Please check if the API key exists manually.",
+  "driver.apiKey.error.nameTooLong": "The name for API key is too long. The maximum length is 128.",
+  "driver.apiKey.error.clientSecretInvalid": "Client secret is invalid. Please make sure: 1. Max %d secrets per API key. 2. Use comma to separate secrets, 3. Length of every secret >= 10 and <= 128",
+  "driver.apiKey.error.domainInvalid": "Domain is invalid. Please make sure provided domain for API key: 1. Max %d domain per API key. 2. Use comma to separate domains",
```

Comment:
Please make sure provided domain for API key: -> Domain for API key should follow:

Reply:
Updated, thanks!